superset/db_engine_specs.py

```diff
@@ -984,14 +984,22 @@ def convert_to_hive_type(col_type):
             return tableschema_to_hive_types.get(col_type, 'STRING')
 
         table_name = form.name.data
+        schema_name = form.schema.data
```

Comment:
Was this just never used? It seems like it's been in the form for several months. Per the original exception it seems like the plan was to enforce uploading to a pre-defined schema.

Reply:
It wasn't. Historically, you could only specify the namespace by making the table name `namespace.tablename`.

The exception happens when the administrator forces all tables to be in one namespace. And in that case, any avenue used to specify a namespace (schema form field or namespace before table name) will lead to an exception.